pyvista/utilities/helpers.py

```diff
@@ -1447,6 +1447,11 @@ def cubemap(path='', prefix='', ext='.jpg'):
     pyvista.Texture
         Texture with cubemap.
 
+    Notes
+    -----
+    Cubemap will appear flipped relative to the XY plane between VTK v9.1 and
+    VTK v9.2.
```

Comment:
Yikes... We should remember to also note this in https://github.com/pyvista/pyvista/pull/2729. Do you know if this was intentional? If it's a bug perhaps it won't make it into 9.2.0, or will be reverted by 9.2.1...

Reply:
Oof. This seems like a bug to me.

Reply:
> Oof. This seems like a bug to me.

@banesullivan, do you want to raise an issue or would you like me to?